<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>521</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>실습실 이용 시간 체크</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Desktop/동아리 운영 관련/스마클~1/스마클 .jpg</normaloff>../../../Desktop/동아리 운영 관련/스마클~1/스마클 .jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;맑은 고딕&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>로그아웃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Desktop/캡처.JPG</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0);
font: 75 15pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>로그아웃을 누르면 PC가 자동으로 종료됩니다. &lt;p&gt;모든 작업이 완료 된 후 로그아웃을 하세요.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelShowTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Current Time :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ShowTime">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 28pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelShowTime_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;PC :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ShowPC">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 28pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelShowTime_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;위치 :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ShowLoc">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>28</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 28pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>현재 접속자:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoginID">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>접속 시간 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoginTime">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>btnLogout</zorder>
   <zorder>label_2</zorder>
   <zorder>labelShowTime</zorder>
   <zorder>labelShowTime_3</zorder>
   <zorder>labelShowTime_4</zorder>
   <zorder>ShowTime</zorder>
   <zorder>ShowLoc</zorder>
   <zorder>ShowPC</zorder>
   <zorder>label_3</zorder>
   <zorder>LoginID</zorder>
   <zorder>label_5</zorder>
   <zorder>LoginTime</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
